test/timearray.jl

```diff
@@ -189,7 +189,7 @@ end
 
 @testset "construct with NamedTuple" begin
     data = (datetime=[DateTime(2018, 11, 21, 12, 0), DateTime(2018, 11, 21, 13, 0)], col1=[10.2, 11.2], col2=[20.2, 21.2], col3=[30.2, 31.2])
-    ta = TimeArray(data, :datetime; meta="Example")
+    ta = TimeArray(data; timestamp=:datetime, meta="Example")
```

Comment:
well, you add the default value back.
I want no default value: https://github.com/JuliaStats/TimeSeries.jl/pull/395#discussion_r236046606